contracts/external/aave/AaveDataTypes.sol

```diff
@@ -0,0 +1,265 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
are `contracts/external/aave/*` only used for fork tests? if yes, i'd avoid creating interfaces and would only declare the abi directly in js. we have over a hundred contracts and if we keep adding the compilation time will grow

Reply:
Yes they are, I'll use their abis instead. There we other external interfaces that's why they were added this way...

Reply:
I think the other interfaces are actually used in the code as well, not just in tests